src/test/java/com/microsoft/sqlserver/jdbc/connection/ConnectionDriverTest.java

```diff
@@ -332,7 +330,7 @@ public void testDeadConnection() throws SQLException {
                 assertEquals(e.getMessage(), TestResource.getResource("R_connectionReset"),
                         TestResource.getResource("R_unknownException"));
             }
-            assertEquals(conn.isValid(5), false, TestResource.getResource("R_deadConnection"));
+            assertEquals(conn.isValid(10), false, TestResource.getResource("R_deadConnection"));
```

Comment:
Do we still need this change after fixing the hanging issue.

Reply:
I'm gonna reset all other changes.. just a sec